proposals/0304-structured-concurrency.md

```diff
@@ -164,19 +164,19 @@ Of course, a function’s task may itself be a child of another task, and its pa
 
 In this proposal, the way to create child tasks is only within a `TaskGroup`, however there will be a follow-up proposal that enables creation of child tasks in any asynchronous context.
 
-### Partial tasks
+### Jobs
```

Comment:
Not sure if we actually want this rename.

I'll double check.

Reply:
> Not sure if we actually want this rename.

@ktoso https://github.com/apple/swift-evolution/pull/1368#discussion_r637101407

Reply:
Ah ok, great then :)